rudder-web/src/main/scala/com/normation/rudder/web/rest/RestUtils.scala

```diff
@@ -94,7 +94,7 @@ object RestUtils extends Loggable {
    *   - Add a new line at the end and beginning of an array and indent one more level array data
    *   - space after colon
    */
-  def render(value: JValue): Document = {
+  def rendrrer(value: JValue): Document = {
```

Comment:
the code is really taken from the given version, and I don't know if it changed or not. Changing the comment without changing the code accordingly would be false, and misleading if someone want to refactor it latter on.